### ISSUE TITLE
Is sourcemap traceback not available for single bundle file

### ISSUE THREAD

**Issue by RobinCodeX** [TypeScriptToLua/TypeScriptToLua#907]
My tsconfig.json
```json
{
    "compilerOptions": {
        "rootDir": "src",
        "target": "esnext",
        "lib": ["esnext"],
        "moduleResolution": "node",
        "types": [
            "lua-types/jit"
        ],
        "strict": true
    },
    "tstl": {
        "luaTarget": "JIT",
        "luaBundle": "ts.lua",
        "luaBundleEntry": "src/main.ts",
        "sourceMapTraceback": true,
        "noImplicitSelf": true,
    }
}
```
After compile:
![image](https://user-images.githubusercontent.com/8408783/88473824-e9cd4300-cf53-11ea-9eea-2e807aa700e0.png)

**Comment by RobinCodeX** [comment]
I updated typescript-to-lua to **0.35.0**, and the same config as above.
After rebuild my project, all library names are wrong, `.src.` is added in front of each file.

![image](https://user-images.githubusercontent.com/8408783/88479373-53b01180-cf81-11ea-8e2d-2d0e40523c03.png)
![image](https://user-images.githubusercontent.com/8408783/88479518-3def1c00-cf82-11ea-90b5-ada55089f66b.png)

**Comment by codecat** [comment]
I think you have to do `"rootDir": ".",` instead of `"src"`.

**Comment by RobinCodeX** [comment]
> I think you have to do `"rootDir": ".",` instead of `"src"`.

It's still the bug, it can't be solved.

**Comment by Perryvw** [comment]
Sourcemap traceback currently does not work in combination with bundling. Bug here is that it should give a clear message, and not leave the `{#SourceMapTraceback}` in there.

**Comment by Perryvw** [comment]
For the bundle paths see #823

**Comment by ark120202** [comment]
This issue with bundle paths is new (and different from #823). It was introduced in https://github.com/TypeScriptToLua/TypeScriptToLua/pull/885/files#diff-aaff76179ed6e10c75c5fa07932dec6cR68:
```ts
const entryPoint = `return require(${createModulePath(outDir, resolvedEntryModule)})\n`;
```
is supposed to be
```ts
const entryPoint = `return require(${createModulePath(rootDir, resolvedEntryModule)})\n`;
```

This would be resolved as a part of #909, but if someone wants to make a smaller PR resolving this exact issue that would be fine too.

**Comment by RobinCodeX** [comment]
> This issue with bundle paths is new (and different from #823). It was introduced in https://github.com/TypeScriptToLua/TypeScriptToLua/pull/885/files#diff-aaff76179ed6e10c75c5fa07932dec6cR68:
> 
> ```ts
> const entryPoint = `return require(${createModulePath(outDir, resolvedEntryModule)})\n`;
> ```
> 
> is supposed to be
> 
> ```ts
> const entryPoint = `return require(${createModulePath(rootDir, resolvedEntryModule)})\n`;
> ```
> 
> This would be resolved as a part of #909, but if someone wants to make a smaller PR resolving this exact issue that would be fine too.

OK, I delete the outDir field, the library names are normal.
